data/style.css

```
@@ -51,6 +59,13 @@ img[alt="XMR Logo"] {
 	body {
 		background: white ;
 		color: black ;
+		scrollbar-color: gray white;
+	}
+	::-webkit-scrollbar {
+		background-color: white;
+	}
+	::-webkit-scrollbar-thumb {
+		background-color: gray;
 	}
```

```suggestion
	}
	::-webkit-scrollbar-track:vertical:disabled:corner-present {
        margin-bottom: 0px;
    }
```
Might you please put this in also, so that when there is no horizontal scrollbar (only would happen when you're really zoomed in) the corner will not be there.